dotcom-rendering/src/components/ArticlePage.tsx

```diff
@@ -41,6 +41,44 @@ interface AppProps extends BaseProps {
 	renderingTarget: 'Apps';
 }
 
+/**
+ * @description
+ * returns global styles for article pages
+ */
+const globalStyles = (
+	format: ArticleFormat,
+	darkModeInApps: boolean | undefined,
```

Comment:
I think this sould be a required argument.

```suggestion
	darkModeInApps: boolean,
```